packages/cli/src/index.ts

```diff
@@ -14,6 +25,26 @@ program
     initCommand();
   });
 
+program
+  .command('sync')
+  .option('-a, --api-url <apiUrl>', 'The Novu Cloud API url, defaults to https://api.novu.co', 'https://api.novu.co')
```

Comment:
Keeping language consistent with the `dev` command [here](https://github.com/novuhq/novu/pull/5860/files#diff-2fb448d11702324cc59d657d5945d59cd325540eb87b71f2e66a5bc01ada834cR24)

```suggestion
  .option('-a, --api-url <apiUrl>', 'The Novu Cloud API url, 'https://api.novu.co')
```